.github/workflows/build.yml

```diff
@@ -6,15 +6,17 @@ env:
 on: [push]
 
 jobs:
-  build:
+  build-linux:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout code
+        uses: actions/checkout@v4
 
       - name: Install Rust
-        uses: dtolnay/rust-toolchain@stable
+        uses: actions-rs/toolchain@v1
```

Comment:
Actually, [dtolnay](https://github.com/dtolnay/rust-toolchain) is the newer action. Repo [actions-rs](https://github.com/actions-rs/toolchain) is archived and last updated 4 years ago.